dulwich/tests/compat/server_utils.py

```diff
@@ -237,6 +237,54 @@ def test_fetch_full_depth_into_shallow_clone_from_dulwich(self):
         self.assertEqual([], _get_shallow(clone))
         self.assertReposEqual(clone, self._source_repo)
 
+    def test_fetch_from_dulwich_issue_88_standard(self):
```

Comment:
FWIW the compat tests are only supposed to test compatibility with C git, they're not the right place for integration tests, which should probably be in dulwich/tests/test_server.py